src/utils/search/topic-extractor.ts

```diff
@@ -1,8 +1,11 @@
 import {SearchState} from 'react-instantsearch-core'
-import {first} from 'lodash'
+import {first, isEmpty} from 'lodash'
 
 export const topicExtractor = (searchState: SearchState) => {
-  const selectedTopics = searchState.refinementList?._tags || []
```

Comment:
`_tags` is always an empty array so what this was doing was using that array and pushing the query topic onto it. Now if the `tags` array has stuff in it we just return it because we don't want to mess with the "query as a topic" and instead just move on. If no tags are explicitly defined we push the single query onto the "selectedTopics" array to display that topic landing page